dietpi/dietpi-software

```diff
@@ -11652,6 +11659,43 @@ _EOF_
 			G_EXEC chmod +x /usr/local/bin/restic
 		fi
 
+		if To_Install 141 # MediaWiki
+		then
+			# Install required PHP modules: https://www.mediawiki.org/wiki/Manual:Installation_requirements#PHP
+			aDEPS=("php$PHP_VERSION-intl")
+			# - Add JSON module for PHP7, as it does not exist (embedded in core package) on PHP8
+			local json=()
+			[[ $PHP_VERSION == 8* ]] || aDEPS+=("php$PHP_VERSION-json") json=('json')
+
+			# Create MariaDB database and user
+			if [[ -d '/mnt/dietpi_userdata/mysql/mediawiki' ]]
+			then
+				G_DIETPI-NOTIFY 2 'MediaWiki MariaDB database found, will NOT overwrite.'
+			else
+				/boot/dietpi/func/create_mysql_db mediawiki mediawiki "$GLOBAL_PW"
+			fi
+
+			# Get latest version
+			local version="1.39"
+			local file="mediawiki-1.39.3"
+			local fallback_url="https://releases.wikimedia.org/mediawiki/1.39/mediawiki-1.39.3.tar.gz"
+			Download_Install "https://releases.wikimedia.org/mediawiki/$version/$file.tar.gz"
+
+			# Reinstall: Clean install but preserve existing config file
+			[[ -f '/var/www/wiki/LocalSettings.php' ]] && G_EXEC mv /var/www/wiki/LocalSettings.php "$file/"
+			[[ -d '/var/www/wiki' ]] && G_EXEC rm -R /var/www/wiki
+
+			# Move new instance in place
+			G_EXEC cp -a "$file/." /var/www/wiki/
+			G_EXEC rm -R "$file"
+
+			unset -v version file
```

Comment:
```suggestion
			unset -v version
```
Okay, as `$file` was used quite often, probably not a bad idea to define it as well, but based on `$version` so that only a single variable needs to be updated manually.